### ISSUE TITLE
Object.dirname is deprecated.

### ISSUE THREAD

**Issue by ljsinclair** [dundalek/atom-markdown-mindmap#43]
Argument to `path.dirname` must be a string
```
Object.dirname (C:\Users\gothe\AppData\Local\atom\app-1.28.0\resources\app.asar\src\electron-shims.js:9:10)
transformLinks (C:\Users\gothe\.atom\packages\markdown-mindmap\lib\markdown-mindmap-view.coffee:42:16)
MarkdownMindmapView.parseMarkdown (C:\Users\gothe\.atom\packages\markdown-mindmap\lib\markdown-mindmap-view.coffee:398:32)
MarkdownMindmapView.renderMarkdownText (C:\Users\gothe\.atom\packages\markdown-mindmap\lib\markdown-mindmap-view.coffee:403:19)
<unknown> (C:\Users\gothe\.atom\packages\markdown-mindmap\lib\markdown-mindmap-view.coffee:242:26)
<unknown> (null:null:null)
```

**Comment by hueri** [comment]
Also seeing this - bummer, as it worked in a previous version!

**Comment by dundalek** [comment]
I was able to give it a look today. I cannot reproduce the issue, so it is probably a bug in the parser which is probably caused by some particular markdown. It would be helpful if you could share the markdown content with which you encounter this issue (make sure to redact out any private and sensitive content before sharing).

**Comment by hueri** [comment]
i just tried again (reverting the change), and i’m seeing what you’re seeing (functional, but with the error).  Previously, it wasn’t opening the mindmap-markdown window, and would only bring up the error console.

The markdown test had been super-simple.
# test
## stuff here
### other

i’ll leave it to you for next steps - thanks for even looking!



On Aug 11, 2019, at 8:28 AM, Jakub Dundalek <notifications@github.com<mailto:notifications@github.com>> wrote:


I was able to give it a look today. I cannot reproduce the issue, so it is probably a bug in the parser which is probably caused by some particular markdown. It would be helpful if you could share the markdown content with which you encounter this issue (make sure to redact out any private and sensitive content before sharing).

**Comment by dundalek** [comment]
Should be fixed in the latest version `0.6.2`.
